plugins/event_source/alertmanager.py

```diff
@@ -2,18 +2,32 @@
 alertmanager.py
 
 An ansible-rulebook event source module for receiving events via a webhook from
-alertmanager.
+alertmanager or alike system.
 
 Arguments:
-    host: The hostname to listen to. Set to 0.0.0.0 to listen on all
+    host: The webserver hostname to listen to. Set to 0.0.0.0 to listen on all
           interfaces. Defaults to 127.0.0.1
     port: The TCP port to listen to.  Defaults to 5000
+    data_alerts_path: The json path to find alert data. Default to "alerts"
+                      Use empty string "" to treat the whole payload data as
+                      one alert.
+    data_host_path: The json path inside the alert data to find alerting host.
+                    Default to  "labels.instance".
```

Comment:
```suggestion
    data_host_path: The json path inside the alert data to find alerting host.
                    Use an empty string `""` to disable the host filtering
                    Default to  "labels.instance".
```

Reply:
no need to set empty string if no host can be found. The code automatically skip it.

Reply:
The default value is `labels.instance` but the user could want to avoid matching any host at all. This can be done setting the host filtering to empty string `""` and that is why I suggest specifying it in the documentation.